client/src/assets/stylesheets/partials/_lesson.scss

```diff
@@ -16,6 +16,15 @@ $letterSize: 8em;
   background: $progress-color;
 }
 
+.lesson-end {
+  text-align: center;
+  margin-top: 1rem;
+}
+
+.lesson-end .btn {
+  background-color: blue;
```

Comment:
This is a low-priority and infrequent user action, we should be more subtle about it to keep the user's focus on the task at hand: the lesson itself.

Reply:
Got it!. I just realized we don't use this code as well.